pkg/plugins/resources/postgres/store.go

```diff
@@ -273,6 +279,20 @@ func rowToItem(resources model.ResourceList, rows *sql.Rows) (model.Resource, er
 	return item, nil
 }
 
+func (r *postgresResourceStore) countRows(resource string) (int, error) {
+	var count int
+	statement, err := r.db.Prepare("SELECT COUNT(*) as count FROM resources WHERE type=" + resource)
```

Comment:
We also need `mesh` here. Listing by type and mesh is the most common case. When you do `kumactl get dataplanes` you actually list dataplanes in `default` mesh. You can do `kumactl get dataplanes --mesh=demo`

Reply:
That was quite a preliminary code. Should be fixed now, thanks!